src/styles/nav.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: $max-width;
+  padding: 0 50px;
+  margin: auto;
+
+  &__logo {
+    margin: 10px 0;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__link {
+    color: $primary-color;
+    display: block;
+    align-items: center;
+    position: relative;
+    margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children
So add margin to nav__item
```suggestion
```